# Big-Data-Lab2

Now, the lab is working without issues
